clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,30 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model http://www.javadoc.io/doc/io.fabric8/kubernetes-model/1.0.73
+//which will map to Kubernetes 1.5 API version
+class KubernetesPodSpecDescription {
+  String name
+  String kind
```

Comment:
Yes.  I can remove them.  I am just sure if we ever need it in podSpec.  https://godoc.org/k8s.io/kubernetes/pkg/api#PodSpec is GO.  I am referring fabric8 for coding purpose.  At the end, both API should have the same fields.